### ISSUE TITLE
Error in 'conv_utils.normalize_data_format' 

### ISSUE THREAD

**Issue by animeshseemendra** [gaudetcj/DeepQuaternionNetworks#6]
While running the code in quaternion layers conv.py file has an error.

**Comment by gaudetcj** [comment]
Hi,

I have not used Keras since this was uploaded. I have switched to Pytorch and have quaternion and generalized hypercomplex network code in that framework.

I will no longer be updating this repo.

If you want to run quaternion or generalized vector map convolutions use my latest repo.
